packages/atomic/src/components/search/facets/atomic-segmented-facet/atomic-segmented-facet.tsx

```diff
@@ -265,7 +309,7 @@ export class AtomicSegmentedFacet implements InitializableComponent {
     return (
       <div
         part="segmented-container"
-        class="flex whitespace-nowrap h-10 items-center"
+        class="flex items-center h-10 whitespace-nowrap"
```

Comment:
Why the reorder

Reply:
I happen to have [Headwind](https://marketplace.visualstudio.com/items?itemName=heybourn.headwind) installed! It sorts the tailwind classes in a consistent order. A better solution is to use the [tailwind prettier plugin](https://tailwindcss.com/blog/automatic-class-sorting-with-prettier)